### PR TITLE
Graph2: Naming Clarifications and Example

### PR DESCRIPTION
**Pull request by pertrai1**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ ] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [x] Documentation change?

Made a couple changes:

1. Explicit names for clarification of intent and better align with the names used in Graph and Graph3
2. `printGraph` string interpolation for readability
3. Example usage

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized. **(chatgpt assistance)**
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in their comments that points to Wikipedia or another similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Data-Structures/Graph/Graph2.js [hunk 1]
```diff
@@ -21,42 +21,49 @@ class Graph {
...
-    this.AdjList.set(v, [])
+    this.adjacencyList.set(v, [])
   }
 
   // add edge to the graph
-  addEdge(v, w) {
-    // get the list for vertex v and put the
-    // vertex w denoting edge between v and w
-    this.AdjList.get(v).push(w)
+  addEdge(vertex1, vertex2) {
```
Comment (appgurueu) [inline Data-Structures/Graph/Graph2.js:28]: `v` and `w` are typical names for vertices. `vertex1` and `vertex2` is less readable.

File: Data-Structures/Graph/Graph2.js [hunk 2]
```diff
@@ -21,42 +21,49 @@ class Graph {
...
 export { Graph }
+
+// Example of usage:
+const graph = new Graph(5);
+graph.addVertex("A");
+graph.addVertex("B");
+graph.addVertex("C");
+graph.addEdge("A", "B");
+graph.addEdge("A", "C");
+graph.printGraph();
```
Comment (appgurueu) [inline Data-Structures/Graph/Graph2.js:65]: No side effects. Don't just put an example here. Put it in a JSDoc `@example` or in the tests.

File: Data-Structures/Graph/Graph2.js (hunk 1 above)
Comment (pertrai1) [reply Data-Structures/Graph/Graph2.js:28]: Would it be best to close this out and leave the file with naming as is? There could be better things to focus on with your time. I am game for that if you think all is good right now.

Comment (appgurueu) [reply Data-Structures/Graph/Graph2.js:28]: It's not major, but it is a minor change for the worse, IMO.

Comment (pertrai1) [reply Data-Structures/Graph/Graph2.js:28]: Ok. Thanks for the feedback @appgurueu

**Pull request closed without merging**